variables.tf

```diff
@@ -348,3 +348,15 @@ variable "exec_enabled" {
   description = "Specifies whether to enable Amazon ECS Exec for the tasks within the service"
   default     = false
 }
+
+variable "deployment_circuit_breaker" {
```

Comment:
@flywheelnz to go with the general naming convention I'd suggest `circuit_breaker_deployment_enabled`

Reply:
Done!